docs/pages/access-controls/sso/oidc.mdx

```diff
@@ -35,112 +32,123 @@ documented on the identity providers website. Here are a few links:
-```
-
-Create the connector:
-
-<ScopedBlock scope={["oss", "enterprise"]}>
+To create a new connector, use `tctl sso configure`. Our example creates a
+connector resource file in YAML format named `oidc-connector.yaml`:
 
 ```code
-# Log in to your cluster with tsh so you can use tctl from your local machine.
-# You can also run tctl on your Auth Service host without running "tsh login"
-# first.
-$ tsh login --proxy=teleport.example.com --user=myuser
-$ tctl create oidc-connector.yaml
+$ tctl sso configure oidc --name <CONNECTOR-NAME> \
+  --issuer-url <PATH-TO-PROVIDER> \
+  --id <CLIENT-ID> --secret <CLIENT-SECRET> \
+  --claims-to-roles <CLAIM-KEY>,<CLAIM-VALUE>,access \
+  --claims-to-roles <CLAIM-KEY>,<CLAIM-VALUE>,editor > oidc-connector.yaml
 ```
 
-</ScopedBlock>
-<ScopedBlock scope={["cloud"]}>
+- `--name`: Usually the name of the IdP, this is how the connector will be
+  identified in Teleport.
+- `--issuer-url`: This is the base path to the IdP's OIDC configuration endpoint,
+  excluding `.well-known/openid-configuration`. If, for example, the endpoint
+  is `https://example.com/.well-known/openid-configuration`, you would use
+  `https://example.com`.
+- `--id`: The client ID as defined in the IdP. Depending on your identity
+  provider this may be something you can define (i.e. `teleport`), or may be an
+  assigned string.
+- `--secret`: The client token/secret provided by the IdP to authorize this client.
+- `--claims-to-roles`: A mapping of OIDC claims/values to be associated with
+  Teleport roles.
+
+For more information on these and all available flags, see the [tctl sso configure
+oidc](docs/reference/cli.mdx#tctl-sso-configure-oidc) section of the Teleport CLI
+Reference page.
+
+The file created should look like the example below. This connector requests
+the scope `<CLAIM-KEY>` from the identity provider, then maps the value to
+either the `access` or the `editor` role depending on the value returned for
+that key within the claims:
 
-```code
-# Log in to your Teleport cluster so you can use tctl remotely.
-$ tsh login --proxy=mytenant.teleport.sh --user=myuser
-$ tctl create oidc-connector.yaml
+```yaml
+(!examples/resources/oidc-connector.yaml!)
 ```
 
-</ScopedBlock>
-
-## Create Teleport Roles
-
-The next step is to define Teleport roles. They are created using the same
-`tctl` [resource commands](../../reference/resources.mdx) as we used for the auth
-connector.
-
-Below are two example roles that are mentioned above, the first is an admin
-with full access to the system while the second is a developer with limited
-access.
+<Details title="Practical Example: Keycloak">
+The following example was generated using Keycloak as the identity provider.
+Keycloak is being served at `keycloak.example.com`, and the Teleport Proxy
+Service is listening at `teleport.example.com`. In Keycloak, the client is
+named `teleport`, and under the `teleport-dedicated` client scope we've added
+the "Group Membership" mapper:
```

Comment:
```suggestion
named `teleport`. Under the `teleport-dedicated` client scope, we've added
the "Group Membership" mapper:
```
I don't really understand this-probably just missing context—but I don't know where ``teleport-dedicated` client scope` is or where `the "Group Membership" mapper` is.

Reply:
Yes, all the context for this lives in the Keycloak UI. I added it because Keycloak's UI is unintuitive IMO and this instruction will help them navigate where they need to do. If Keycloak were more popular a dedicated guide might be in order.